burger.css

```diff
@@ -0,0 +1,58 @@
+   
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)